src/Okta.Sdk/AddCustomHotpFactorOptions.cs

```diff
@@ -0,0 +1,27 @@
+// <copyright file="AddPasswordPolicyRuleOptions.cs" company="Okta, Inc">
+// Copyright (c) 2014 - present Okta, Inc. All rights reserved.
+// Licensed under the Apache 2.0 license. See the LICENSE file in the project root for full license information.
+// </copyright>
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Okta.Sdk
+{
+    /// <summary>
+    /// Custom HMAC-based One-Time Password (HOTP) factor options
+    /// </summary>
+    public class AddCustomHotpFactorOptions
```

Comment:
Is this the established naming convention?  Specifically wondering about the verb prefix **Add**CustomHotpFactorOptions vs just CustomHotpFactorOptions.